DevEdu.API/DevEdu.DAL/Repositories/CourseRepository.cs

```diff
@@ -18,8 +18,9 @@ public CourseRepository()
 
         public void AddTagToTopic(int topicId, int tagId)
         {
+            string query = "dbo.Tag_Topic_Insert";

```

Comment:
please move all the stored procedures' names to the top of the class (like it was in the lesson)